### PR TITLE
docs: configuring app names

### PR DESCRIPTION
**Pull request by DmitrDomrachev**
## Changes
- added instructions for configuring the names of different falvors builds

## Checklist for self-check
- [x] Commits and PRs have been filed according to [the rules on the project](https://github.com/surfstudio/surf-flutter-app-template#workflow-in-a-repository).
- [x] The author is marked as an assigne and assigned mandatory reviewers.
- [x] Required labels marked
- [x] Specified related tasks and/or related PRs.
- [x] Specified Changes.
- [ ] Attached videos/screenshots demonstrating the fix/feature.
- [x] All unspecified fields in the PR description deleted.
- [ ] New code covered by tests.

## Checklist for reviewers
- [ ] CI passed successfully _(with a green check mark)_.
- [ ] PR is atomic, by volume no more than 400 (+-) corrected lines (not including codogen).

Design:
- [ ] System design corresponds to the agreements on structure and architecture on the project.
- [ ] The code is decomposed into necessary and sufficient components.

Functionality:
- [ ] The code solves the problem.
- [ ] Any changes to the user interface are reasonable and look good.

Complexity:
- [ ] The code is clear, easy to read, functions are small, no more than 50 lines.
- [ ] The logic is not overcomplicated, there is no overengineering (no code sections that may be needed in the future, but no one knows about it).

Tests:
- [ ] Updated or added tests for mandatory components.
- [ ] The tests are correct, helpful, and well designed/developed.

Naming:
- [ ] The naming of variables, methods, classes and other components is understandable.

Comments:
- [ ] The comments are understandable and helpful.

Documentation:
- [ ] All labels are correct
- [ ] Technical documentation updated (after approval, updates last reviewer).

### CODE REVIEW

File: docs/flavors.md [hunk 1]
```diff
@@ -2,50 +2,52 @@
...
+        versionNameSuffix "-dev"
+    }
+    prod {
+        dimension "release-type"
+    }
+    qa {
+        dimension "release-type"
+        applicationIdSuffix ".qa"
+        versionNameSuffix "-qa"
+    }
```
Comment (dzmitry-struk-surf) [inline docs/flavors.md:43]: We do not have `qa` flavor anymore in our template anymore. 
https://github.com/surfstudio/surf-flutter-app-template/pull/100

File: docs/flavors.md [hunk 2]
```diff
@@ -2,50 +2,52 @@
...
+}
 ```
 
-You also can create directory for your flavor in [`android/app/src`](android/app/src) directory. 
+You also can create directory for your flavor in [`android/app/src`](android/app/src) directory.
 
-For example, if you have files that are used only in your flavor (e.g. launch icons, splash, app name, etc), you can create directory `qa` in `android/app/src` and put your files there.
+For example, if you have files that are used only in your flavor (e.g. launch icons, splash, app
+name, etc), you can create directory `qa` in `android/app/src` and put your files there.
```
Comment (dzmitry-struk-surf) [inline docs/flavors.md:50]: Same as above.

File: docs/flavors.md [hunk 3]
```diff
@@ -73,19 +76,28 @@ For example, if you have files that are used only in your flavor (e.g. launch ic
 
 > Replace `qa` with your flavor name.
 
-4. Go to `Runner -> Project -> Runner -> Info` and duplicate configurations (Debug, Release, Profile (if you need it)) for your new flavor. Postfix for your new configuration should be the same as your flavor name. In our case it is `qa`. So, we'll have `Debug-qa`, `Release-qa`, `Profile-qa`. You also should set correct configuration files for each of your new configuration. In the end you should have something like this:
+4. Go to `Runner -> Project -> Runner -> Info` and duplicate configurations (Debug, Release,
+   Profile (if you need it)) for your new flavor. Postfix for your new configuration should be the
+   same as your flavor name. In our case it is `qa`. So, we'll
+   have `Debug-qa`, `Release-qa`, `Profile-qa`. You also should set correct configuration files for
+   each of your new configuration. In the end you should have something like this:
```
Comment (dzmitry-struk-surf) [inline docs/flavors.md:83]: Same as above.

File: docs/flavors.md [hunk 4]
```diff
@@ -101,13 +113,13 @@ Open `.vscode/launch.json` and add the following code:
...
-      "--flavor",
-       "qa"
-   ]
+  "name": "Run qa",
+  "request": "launch",
+  "type": "dart",
+  "args": [
+    "--flavor",
+    "qa"
+  ]
```
Comment (dzmitry-struk-surf) [inline docs/flavors.md:122]: Same as above.

**Review by dzmitry-struk-surf — CHANGES_REQUESTED** [review]
In general - LGTM.

**Review by Sadhorsephile — APPROVED** [review]

**Pull request merged**